docs/al24_slide_000.md

```diff
@@ -1,7 +1,7 @@
 # Non-Platonic Autopoiesis of a Cellular Automaton Glider in Asymptotic Lenia
 
 {:style="text-align:center;"}
-![teaser figure showing instability landscape for asymdrop CA glider](https://raw.githubusercontent.com/riveSunder/fractal_persistence/master/docs/assets/asymdrop_asymdrop_dt_mu.png)
+![teaser figure showing instability landscape for asymdrop CA glider](https://raw.githubusercontent.com/riveSunder/fractal_persistence/master/docs/assets/asymdrop_dt_mu.png)
 
```

Comment:
name  | mp4 | png
vid0a  | mp4 already in project, png thumbnail missing. 
vid1a  |   ✓   | (already in project)
vid1b  |   ✓   |   ✓   
vid1c  |    ✓  |   ✓